rpcserver.go

```diff
@@ -1152,11 +1152,18 @@ func (r *rpcServer) VerifyProof(ctx context.Context,
 	}
 	valid := err == nil
 
-	decodedProof, err := r.marshalProofFile(ctx, proofFile, 0, false, false)
+	p, err := proofFile.ProofAt(uint32(proofFile.NumProofs() - 1))
```

Comment:
We could use `proofFile.LastProof()` here.

Reply:
Totally, just forgot we had that. Updated.